prepare_cloud_image.py

```diff
@@ -83,8 +84,35 @@ def build(self):
         self.download()
         self.prepare_disk_image()
         self.create_cloud_init_image()
+        self.create_kitchen_yml()
         self.run_qemu()
 
+    def create_kitchen_yml(self, name, extra = ()):
+        with open('.kitchen.yml', 'w') as f:
+            print(f)
+            f.write(
+                'driver:\n' +
```

Comment:
Can we extract this as a template string using triple-quote string syntax? e.g. [this hostapd.conf template](https://github.com/liquidinvestigations/setup/blob/master/ansible/roles/hotspot/templates/boot.py#L11-L30)